msfconsole

```diff
@@ -13,6 +13,7 @@ begin
     Warning[:deprecated] = false
   end
 
+  $LOAD_PATH.unshift(Pathname.new(__FILE__).realpath.expand_path.parent)
```

Comment:
Is it worth double checking if we'll need additional lines in the other script entrypoints? 👀